_site/plague-year/index.html

```diff
@@ -145,6 +145,8 @@
             
             <a href="/itinerary/melville-in-rome/">Map: Melville in Rome</a>
             
+            <a href="/itinerary/melville-in-london/">Map: Melville in London</a>
```

Comment:
Nav could be built by looping over maps listed in a manifest on the MEL Cat /maps or /itineraries endpoint